packages/core/integration-tests/test/postcss.js

```diff
@@ -291,7 +291,7 @@ describe('postcss', () => {
         name: 'style.css',
         assets: ['style.css'],
         includedFiles: {
-          'style.css': ['.postcssrc', 'config.css', 'package.json'],
```

Comment:
Shouldn't this still be a dependency?

Just wanna check to ensure there's nothing wrong with the caching on this implementation

Reply:
These will be included files on the Config instead of the Asset now. Can't really test it the same way since you don't have access to the config files. For Babel I added some tests that build, change config, build again and make sure the output changed in the way that I expected.

Reply:
Ow alright, will leave it as is for now then. I'll add additional tests if requested or if issues arise

Reply:
Can the `assertBundles` be extended to assert configs (and these files) as well?

Reply:
Well the config objects are not returned as part of the `BundleGraph` at the moment. We were a bit limited before when core was loading configs in the main process because we didn't have access to the Asset yet, but now we could attach that information to the Asset if we think it would be valuable.